src/lib/components/challenges/ChallengeList.svelte

```diff
@@ -6,7 +6,11 @@
 </script>
 
 <div class="container">
-	{#each challenges as challenge}
-		<ChallengeListItem {challenge} />
-	{/each}
+	<div class="row row-cols-lg-4 row-cols-md-3 row-cols-sm-2">
+		{#each challenges as challenge, idx}
+			<div class="col my-2">
+				<ChallengeListItem challenge={challenges[idx]} />
```

Comment:
why cant we just use challenge instead of looking it up in challenges?

Reply:
Because that would make more sense :)